src/index.html

```diff
@@ -1,18 +1,75 @@
 <!doctype html>
-<html lang="en">
+<html
```

Comment:
The `<html>` tag is self-closing, which is incorrect. HTML tags like `<html>` should not be self-closed. Please remove the self-closing slash to correct this.